npe2/cli.py

```diff
@@ -493,5 +499,28 @@ def cache(
                 typer.secho(f"{mf.name}: {mf.package_version}", fg=typer.colors.GREEN)
 
 
+@app.command()
+def compile(
+    src_dir: str,
+    output: Optional[Path] = typer.Option(
+        None,
+        "-o",
+        "--output",
+        exists=False,
+        help="If provided, will write manifest to filepath (must end with .yaml, "
+        ".json, or .toml). Otherwise, will print to stdout.",
+    ),
+    format: Format = typer.Option(
+        "yaml", "-f", "--format", help="Markdown format to use."
+    ),
+):
+    """Compile napari_plugin_engine plugins."""
```

Comment:
I suspect the description of the command isn't quite right.

I'm a little confused about this.  Is this compiling the `@npe2.implements` stuff or is it extracting npe1 hooks?

`_inspection.compile()` ends up calling `find_npe2_module_contributions()` which seems non-npe1.

Reply:
Correct, this whole implements/compile thing is about autogenerating manifests for npe2 plugins.  Npe1 not supported

Reply:
apologies!  i missed the crux of this comment.  thanks for the correction